assets/constants.js

```diff
@@ -499,12 +499,16 @@ var constants = {
       "content_ids_to_audio_translations": {
           "content": {},
           "default_outcome": {}
+      },
+      "content_translations": {
+          "content": {},
+          "default_outcome": {}
       }
   },
 
   "ALLOW_YAML_FILE_UPLOAD": false,
 
-  "CURRENT_STATES_SCHEMA_VERSION": 25,
+  "CURRENT_STATES_SCHEMA_VERSION": 26,
```

Comment:
Note that @vibhor98's PR is migrating to schema version 26, too. Might want to coordinate.